### PR TITLE
Ahmm I changed some stuff thats cool

### PR DESCRIPTION
**Pull request by KushiKreates**
**I really don't know wht I am doing**

### CODE REVIEW

File: resources/scripts/components/statuspage.tsx
Comment (naterfute) [inline resources/scripts/components/statuspage.tsx:1]: The page title can't be changed from the config file

**Comment by naterfute** [comment]
I also need you to re-add the .github folder, and delete stuff such as .turbot and the other things that were inside the .gitignore file

**Comment by KushiKreates** [comment]
> I also need you to re-add the .github folder, and delete stuff such as .turbot and the other things that were inside the .gitignore file

Everything is fixed, you can set up the status page and stuff via the  php artisan p:environment:setup it will automatically configure everything!

**Comment by naterfute** [comment]
This is going to merge into a side branch for me to fix some issues, after that I will merge with dev, then pr to main

**Pull request merged**